care/utils/sms/backend/sns.py

```diff
@@ -0,0 +1,92 @@
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+
+from care.utils.sms.backend.base import SmsBackendBase
+from care.utils.sms.message import TextMessage
+
+try:
+    import boto3
+    from botocore.exceptions import ClientError
+
+    HAS_BOTO3 = True
+except ImportError:
+    HAS_BOTO3 = False
+
+
+class SnsBackend(SmsBackendBase):
```

Comment:
We can create an instance of the backend class in the settings file itself and re-use it,  we dont have to go through the configuration everytime.